source/rst/dynamic_programming/career.rst

```diff
@@ -322,14 +322,14 @@ Exercise 1
 
 Using the default parameterization in the ``CareerWorkerProblem``,
 generate and plot typical sample paths for :math:`\theta` and :math:`\epsilon`
-when the worker follows the optimal policy
+when the worker follows the optimal policy.
 
 In particular, modulo randomness, reproduce the following figure (where the horizontal axis represents time)
```

Comment:
@AnjuJoon Do we know why this wasn't picked up?

Reply:
hi @arnavs this line is not picked up because it refers to some figure below it for example in this line "the following figure ..."

The script is such that if the line refers to some code below or figure or math equation just next to it then it will not add a full stop after that line. 

If you want that to be changed then it's not a problem, I can do that. Thanks for merging